components/GymPlan/ExerciseSet.js

```diff
@@ -0,0 +1,101 @@
+import styled from "styled-components";
+
+export default function ExerciseSet() {
+  return (
+    <StyledDiv>
+      <input
+        aria-label="set number"
+        type="number"
+        name="set"
+        label="set"
+        placeholder="set"
+        min="0"
+      />
+      <p> x </p>
+      <input
+        aria-label="set volume"
+        type="number"
+        name="volume"
+        label="volume"
+        placeholder="vol"
+        min="0"
+      />
+
+      <input
+        aria-label="set weight"
+        type="number"
+        name="weight"
+        label="weight"
+        placeholder="weight"
+        min="0"
+      />
+      <input
+        aria-label="set exercise"
+        type="text"
+        name="exercise"
+        label="exercise"
+        autoComplete="off"
+        placeholder="exercise"
+      />
+    </StyledDiv>
+  );
+}
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  margin-left: 0;
+  margin-right: 0;
+
+  p {
+    font-size: small;
+    margin-right: 8px;
+  }
+
+  input {
+    height: 30px;
+    border-radius: 5px;
+    border: 1.5px solid lightgray;
+    text-align: center;
+  }
+
+  input[type="number"] {
```

Comment:
You could also make every input a styled component. Then you could also put the type in the component itself instead of using it here in the CSS. But maybe this works better for you :)